modules/distribution/src/repository/resources/dbscripts/apimgt/h2.sql

```diff
@@ -263,112 +265,111 @@ CREATE TABLE IF NOT EXISTS SP_FEDERATED_IDP (
 ALTER TABLE SP_FEDERATED_IDP ADD CONSTRAINT STEP_ID_CONSTRAINT FOREIGN KEY (ID) REFERENCES SP_AUTH_STEP (ID) ON DELETE CASCADE;
 
 CREATE TABLE IF NOT EXISTS SP_CLAIM_MAPPING (
-            ID INTEGER NOT NULL AUTO_INCREMENT,
-            TENANT_ID INTEGER NOT NULL,
-            IDP_CLAIM VARCHAR (512) NOT NULL ,
-            SP_CLAIM VARCHAR (512) NOT NULL ,
-            APP_ID INTEGER NOT NULL,
-            IS_REQUESTED VARCHAR(128) DEFAULT '0',
-            DEFAULT_VALUE VARCHAR(255),
-            PRIMARY KEY (ID));
+	    	ID INTEGER NOT NULL AUTO_INCREMENT,
+	    	TENANT_ID INTEGER NOT NULL,
+	    	IDP_CLAIM VARCHAR (512) NOT NULL ,
+	    	SP_CLAIM VARCHAR (512) NOT NULL ,
+	    	APP_ID INTEGER NOT NULL,
+	    	IS_REQUESTED VARCHAR(128) DEFAULT '0',
+	    	DEFAULT_VALUE VARCHAR(255),
+	    	PRIMARY KEY (ID));
 
 ALTER TABLE SP_CLAIM_MAPPING ADD CONSTRAINT CLAIMID_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE;
 
 CREATE TABLE IF NOT EXISTS SP_ROLE_MAPPING (
-            ID INTEGER NOT NULL AUTO_INCREMENT,
-            TENANT_ID INTEGER NOT NULL,
-            IDP_ROLE VARCHAR (255) NOT NULL ,
-            SP_ROLE VARCHAR (255) NOT NULL ,
-            APP_ID INTEGER NOT NULL,
-            PRIMARY KEY (ID));
+	    	ID INTEGER NOT NULL AUTO_INCREMENT,
+	    	TENANT_ID INTEGER NOT NULL,
+	    	IDP_ROLE VARCHAR (255) NOT NULL ,
+	    	SP_ROLE VARCHAR (255) NOT NULL ,
+	    	APP_ID INTEGER NOT NULL,
+	    	PRIMARY KEY (ID));
 
 ALTER TABLE SP_ROLE_MAPPING ADD CONSTRAINT ROLEID_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE;
 
 CREATE TABLE IF NOT EXISTS SP_REQ_PATH_AUTHENTICATOR (
-            ID INTEGER NOT NULL AUTO_INCREMENT,
-            TENANT_ID INTEGER NOT NULL,
-            AUTHENTICATOR_NAME VARCHAR (255) NOT NULL ,
-            APP_ID INTEGER NOT NULL,
-            PRIMARY KEY (ID));
+	    	ID INTEGER NOT NULL AUTO_INCREMENT,
+	    	TENANT_ID INTEGER NOT NULL,
+	    	AUTHENTICATOR_NAME VARCHAR (255) NOT NULL ,
+	    	APP_ID INTEGER NOT NULL,
+	    	PRIMARY KEY (ID));
 
 ALTER TABLE SP_REQ_PATH_AUTHENTICATOR ADD CONSTRAINT REQ_AUTH_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE;
 
 CREATE TABLE IF NOT EXISTS SP_PROVISIONING_CONNECTOR (
-            ID INTEGER NOT NULL AUTO_INCREMENT,
-            TENANT_ID INTEGER NOT NULL,
-            IDP_NAME VARCHAR (255) NOT NULL ,
-            CONNECTOR_NAME VARCHAR (255) NOT NULL ,
-            APP_ID INTEGER NOT NULL,
-            IS_JIT_ENABLED CHAR(1) NOT NULL DEFAULT '0',
-            BLOCKING CHAR(1) NOT NULL DEFAULT '0',
-            PRIMARY KEY (ID));
+	    	ID INTEGER NOT NULL AUTO_INCREMENT,
+	    	TENANT_ID INTEGER NOT NULL,
+	    	IDP_NAME VARCHAR (255) NOT NULL ,
+	    	CONNECTOR_NAME VARCHAR (255) NOT NULL ,
+	    	APP_ID INTEGER NOT NULL,
+	    	IS_JIT_ENABLED CHAR(1) NOT NULL DEFAULT '0',
+	    	BLOCKING CHAR(1) NOT NULL DEFAULT '0',
+	    	PRIMARY KEY (ID));
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04